### ISSUE TITLE
Backup & restore

### ISSUE THREAD

**Issue by jameswmcnab** [ruckstack/ruckstack#41]
It would be great to have `system-control backup` and `system-control restore` commands available on the installed instance, to copy/restore of volume data on local storage.

Perhaps this is outside the scope of Ruckstack but it's certainly something I find myself wanting to give customers who install software. Perhaps using something like https://github.com/vmware-tanzu/velero?

**Comment by nvoxland** [comment]
Yes, something like that will be great. I'll take a look at the velero and see what they are doing.

I like the idea of just backing up all the volumes, regardless of what is in there. If the application does not need some backed up, they can label them with a ruckstack.backup=false label or something.

**Comment by nvoxland** [comment]
Kubernetes 1.20 ships the stable version of the volume snapshot API
